assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -19,41 +19,87 @@
 .googlesitekit-overlay-notification {
 	background-color: $c-white;
 	border-radius: $br-sm;
-	bottom: 0;
-	box-shadow: 0 8px $grid-gap-desktop 0 rgba(0, 0, 0, 0.25);
-	max-width: 70vw;
-	position: fixed;
-	right: 0;
-	z-index: $z-index-sk-overlay-notification;
+	margin: $grid-gap-phone;
 
 	@media (min-width: $bp-tablet) {
+		margin: $grid-gap-desktop;
+	}
+
+	@media (min-width: $bp-desktop) {
 		bottom: 36px;
+		box-shadow: 0 8px $grid-gap-desktop 0 rgba(0, 0, 0, 0.25);
+		margin: 0;
+		max-width: 70vw;
 		max-width: 296px;
+		position: fixed;
 		right: 36px;
+		z-index: $z-index-sk-overlay-notification;
 	}
 
 	.googlesitekit-overlay-notification__body {
-		padding: $grid-gap-desktop;
-		padding-bottom: 0;
+		color: $c-surfaces-on-surface;
+		padding: $grid-gap-phone;
 
 		h3 {
-			color: $c-neutral-n-500;
-			margin: 0 0 4px;
+			font-size: $fs-title-md;
+			font-weight: $fw-medium;
+			letter-spacing: $ls-xxs;
+			line-height: $lh-title-md;
+			margin: 0 0 10px;
 		}
 
 		p {
-			font-weight: $fw-medium;
-			letter-spacing: 0;
-			margin: 0 0 $grid-gap-desktop;
+			font-size: $fs-body-sm;
+			font-weight: $fw-normal;
+			letter-spacing: $ls-xs;
+			line-height: $lh-body-sm;
+			margin: 0;
+		}
```

Comment:
Maybe we should add new scss mixins for `headline/small`, `title/medium`, `body/medium` etc? For future reuse? Happy to do it in this PR or file a follow up issue.

Reply:
A follow-up issue sounds great!

Reply:
Issue filed for this [here](https://github.com/google/site-kit-wp/issues/8581) :tada: